### DISCUSSION TITLE
Custom Domain on gh-pages not rendering images from react

### DISCUSSION THREAD

**Discussion by MerelJac** [community/community#73279] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey guys! 

So I made a portfolio with react and hosted it on github pages just fine. I purchased a domain name, followed these [instructions](https://medium.com/@madelinecorman/deploying-your-react-app-with-github-pages-and-adding-a-godaddy-custom-domain-name-50a99accd455) to deploy it, and it worked. 

BUT none of my images are rendering (except the icons that are Base64). I have tried moving the files around, renaming the file structure, alllll sorts of things but I am totally at a loss. I think the problem is that, according to the dev tools, my images are coming from `src="/react-portfolio/static/media/github.0f4ef359e26f4435f995.png"`. 

Any advice would be great. Thanks all! 

here is my [repo](https://github.com/MerelJac/react-portfolio/) and the[ domain name](https://mereljac.dev/)

**Comment by GigaSmurf** [comment, marked as answer]
Hey there!

I don't see anything alarming at first glance although, in package.json, maybe change the homepage to `"homepage": "https://mereljac.dev/"`?

**Reply by MerelJac** [reply]
Yes that worked - thank you!

**Reply by GigaSmurf** [reply]
No problem!
